Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2138,8 +2174,9 @@ void CGContextClearRect(CGContextRef context, CGRect rect) {
     ComPtr<ID2D1CommandList> commandList;
     RETURN_IF_FAILED(deviceContext->CreateCommandList(&commandList));
 
-    deviceContext->BeginDraw();
+    EscapeBeginEndDrawStack();
     deviceContext->SetTarget(commandList.Get());
```

Comment:
I think I lied and it doesn't matter [https://msdn.microsoft.com/en-us/library/windows/desktop/hh404533(v=vs.85).aspx](https://msdn.microsoft.com/en-us/library/windows/desktop/hh404533(v=vs.85).aspx)

---
In reply to: [97687941](https://github.com/Microsoft/WinObjC/pull/1750#discussion_r97687941) [](ancestors = 97687941)